wolfcrypt/src/asn.c

```diff
@@ -31117,11 +31117,13 @@ static int EccSpecifiedECDomainDecode(const byte* input, word32 inSz,
     if ((ret == 0) && (version < 1 || version > 3)) {
         ret = ASN_PARSE_E;
     }
+#ifndef WOLFSSL_RELAXED_ASN_PARSING
```

Comment:
Looking at the top of asn.c we have: WOLFSSL_NO_ASN_STRICT
I think that's a better choice.

Reply:
Excellent suggestion! Thank you!!